WowPacketParserModule.V7_0_3_22248/Parsers/GarrisonHandler.cs

```diff
@@ -192,5 +192,75 @@ public static void HandleGetGarrisonInfoResult(Packet packet)
                         ReadGarrisonFollower(packet, "Follower", i, j);
             }
         }
+
+        [Parser(Opcode.SMSG_GARRISON_REQUEST_BLUEPRINT_AND_SPECIALIZATION_DATA_RESULT)]
+        public static void HandleGarrisonRequestBlueprintAndSpecializationDataResult(Packet packet)
+        {
+            packet.ReadInt32("GarrTypeID");
+            var int8 = packet.ReadInt32("SpecializationsKnownCount");
+            var int4 = packet.ReadInt32("BlueprintsKnownCount");
+
+            for (var i = 0; i < int8; i++)
+                packet.ReadInt32("SpecializationsKnown", i);
+
+            for (var i = 0; i < int4; i++)
+                packet.ReadInt32("BlueprintsKnown", i);
+        }
+
+        [Parser(Opcode.SMSG_GARRISON_ADD_FOLLOWER_RESULT)]
+        public static void HandleGarrisonAddFollowerResult(Packet packet)
+        {
+            packet.ReadInt32("GarrTypeID");
+            packet.ReadUInt32("Result");
+            ReadGarrisonFollower(packet);
+        }
+
+        [Parser(Opcode.SMSG_GARRISON_LEARN_BLUEPRINT_RESULT)]
+        public static void HandleGarrisonLearnBlueprintResult(Packet packet)
+        {
+            packet.ReadInt32("GarrTypeID");
+            packet.ReadInt32("Result");
+            packet.ReadInt32("BuildingID");
+        }
+
+        [Parser(Opcode.SMSG_GARRISON_PLACE_BUILDING_RESULT)]
+        public static void HandleGarrisonPlaceBuildingResult(Packet packet)
+        {
+            packet.ReadInt32("GarrTypeID");
+            packet.ReadInt32("Result");
+            ReadGarrisonBuildingInfo(packet, "BuildingInfo");
+
+            packet.ResetBitReader();
+
+            packet.ReadBit("UnkBit");
+        }
+
+        [Parser(Opcode.SMSG_GARRISON_START_MISSION_RESULT)]
+        public static void HandleGarrisonStartMissionResult(Packet packet)
+        {
+            packet.ReadInt32("Result");
+            packet.ReadInt16("UnkInt16"); // increases per Mission
+            ReadGarrisonMission(packet);
+
+            var followerCount = packet.ReadInt32("FollowerCount");
+            for (int i = 0; i < followerCount; i++)
+                packet.ReadInt64("FollowerDBIDs");
+        }
+
+        [Parser(Opcode.SMSG_GARRISON_FOLLOWER_CHANGED_XP)] // GARRISON_FOLLOWER_XP_CHANGED
+        public static void HandleGarrisonUnk2(Packet packet)
+        {
+            packet.ReadUInt32("Result");
+            packet.ReadUInt32("UnkUInt32");
+            ReadGarrisonFollower(packet);
+            ReadGarrisonFollower(packet);
+        }
+
+        [Parser(Opcode.CMSG_OPEN_MISSION_NPC)]
+        public static void HandleGarrisonNpcGUID(Packet packet)
+        {
+            packet.ReadPackedGuid128("NpcGUID");
+            packet.ReadInt32E<GarrisonMissionType>("GarrMissionType");
```

Comment:
this is FollowerTypeID
like this
            Garrison = 1,
            Shipyard = 2,
            Follower = 4